src/components/navbar/MenuList.tsx

```diff
@@ -0,0 +1,118 @@
+              className="md:hover:text-white"
+            />
+          </svg>
+          <span>GitHub</span>
+        </a>
+      </li>
+    </MenuListContainer>
+  )
+}
+
+// STYLED COMPONENTES
+
+const MenuListContainer = styled.ul`
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+  li {
+    list-style: none;
+    a {
+      display: flex;
+      align-items: center;
+      color: ${theme.colors.gray800};
+      font-size: 1.5rem;
+      line-height: 2rem;
+      padding-top: 0.75rem;
+      text-decoration: none;
+      padding-bottom: 0.75rem;
+      font-weight: 500;
+      &.github {
+        svg {
+          margin-right: 0.5rem;
+          path {
+            fill: ${theme.colors.gray800};
+          }
+        }
+      }
+      &:focus {
+        outline: none;
+      }
```

Comment:
Do we want to add some additional styling if we are removing the outline on focus? Keyboard users wont see any difference now

Reply:
100%, re-added the default focus ring for this solution.